CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 
 # Alfred API - Changelog
 
+## 2.7.0 (unreleased)
 
-
-## 2.7.0 (2020-10-01)
+### Changed
+* [ALFREDAPI-461](https://xenitsupport.jira.com/browse/ALFREDAPI-418): Reintroduce xenit maven repo definition
```

Comment:
@RVanhuysseXenit Link is not pointing to the corresponding ticket.

Reply:
good spot. fixed it